public/javascripts/webrtc.js

```diff
@@ -61,13 +64,9 @@ requestUserMedia($self.mediaConstraints);
 /*
 First page forms
 */
-function handleUserNames(event) {
-  event.preventDefault();
-  const form = event.target;
-  const username = form.querySelector('#username-input').value;
-  const figcaption = document.querySelector('#self figcaption');
-  figcaption.innerText = username;
-}
+document.querySelector('#yourusername')
```

Comment:
it's better to move all these form related logic to main.js
try this selector `#JoinForm `